nmostesting/suites/BCP0050201Test.py

```diff
@@ -0,0 +1,474 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# python3 nmos-test.py suite BCP-005-02 --host 127.0.0.1 127.0.0.1 --port 5058 5058 --version v1.3 v1.1
+
+import json
+import re
+import os
+
+from jsonschema import ValidationError
+
+from ..GenericTest import GenericTest, NMOSTestException
+from ..IS04Utils import IS04Utils
+from ..IS05Utils import IS05Utils
+from ..TestHelper import load_resolved_schema
+from ..TestHelper import check_content_type
+
+from pathlib import Path
+
+NODE_API_KEY = "node"
+CONNECTION_API_KEY = "connection"
+RECEIVER_CAPS_KEY = "receiver-caps"
+SENDER_CAPS_KEY = "sender-caps"
+
+# Generic capabilities from any namespace
+
+
+def cap_without_namespace(s):
+    match = re.search(r'^urn:[a-z0-9][a-z0-9-]+:cap:(.*)', s)
+    return match.group(1) if match else None
+
+
+def get_key_value(obj, name):
+    regex = re.compile(r'^urn:[a-z0-9][a-z0-9-]+:' + name + r'$')
+    for key, value in obj.items():
+        if regex.fullmatch(key):
+            return value
+    return obj[name]  # final try without a namespace
+
+
+def has_key(obj, name):
+    regex = re.compile(r'^urn:[a-z0-9][a-z0-9-]+:' + name + r'$')
+    for key in obj.keys():
+        if regex.fullmatch(key):
+            return True
+    return name in obj  # final try without a namespace
+
+
+class BCP0050201Test(GenericTest):
+    """
+    Runs Node Tests covering sender and receiver with IPMX/HKEP
+    """
+
+    def __init__(self, apis, **kwargs):
+        # Don't auto-test /transportfile as it is permitted to generate a 404 when master_enable is false
+        omit_paths = [
+            "/single/senders/{senderId}/transportfile",
+            "/single/senders/{senderId}/staged",
+            "/single/senders/{senderId}/active",
+            "/single/senders/{senderId}/constraints",
+            "/single/senders/{senderId}/transporttype",
+            "/single/receivers/{receiverId}/staged",
+            "/single/receivers/{receiverId}/active",
+            "/single/receivers/{receiverId}/constraints",
+            "/single/receivers/{receiverId}/transporttype",
+        ]
+        GenericTest.__init__(self, apis, omit_paths, **kwargs)
+        self.node_url = self.apis[NODE_API_KEY]["url"]
+        self.connection_url = self.apis[CONNECTION_API_KEY]["url"]
+        self.is04_resources = {"senders": {}, "receivers": {}, "_requested": [], "sources": {}, "flows": {}}
+        self.is05_resources = {"senders": [], "receivers": [],
+                               "_requested": [], "transport_types": {}, "transport_files": {}}
+        self.is04_utils = IS04Utils(self.node_url)
+        self.is05_utils = IS05Utils(self.connection_url)
```

Comment:
Only used by get_is05_partial_resources which is itself unused, so can be removed
```suggestion
```